desc/examples/update_all_equilibria_backwards_compatibility.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
this file might be better placed in `devtools` rather than in the desc source. 

in either case should also add it to `setup.cfg` to have coverage ignore it